### ISSUE TITLE
Defining ChestRange and other pixels

### ISSUE THREAD

**Issue by sfai05** [Reasel/CrushThemAllMonkeyScripts#1]
Would you mind to share more about how to correctly define the constant like `ChestRange` and pixel color ?

**Comment by Reasel** [comment]
Yeah I can try!

So in android you can enable developer options [something like this](https://www.howtogeek.com/129728/how-to-access-the-developer-options-menu-and-enable-usb-debugging-on-android-4.2/) it varies from phone to phone. You need to do this anyway to enable usb debug in order to run the scripts.

One of the options is "Show pointer location" and "Show touches". For me the pointer location shows a horizontal bar and vertical bar that pinpoint where your phone detects you are touching. Also it shows coordinates at the top. I used this to actually map the bounds of the chest area.

Basically just touch the top left of the area you want to set as the chestrange and then note the coords. Repeat for bottom right.

Then you can use the `device.takeSnapshot().getRawPixel(x, y)` where x and y are the coords for the pixel you want to check to get the color of that pixel.

I made it this way due to every device being different. I hope this helps.

**Comment by sfai05** [comment]
@Reasel Thanks for the tips of using "Show pointer location" and "Show touches" ! Its working now. Thanks a lot
